api/room_resource/schema.py

```diff
@@ -226,6 +231,15 @@ def resolve_get_resources_by_room_id(self, info, room_id):
             resources.append(room_resource)
         return RoomResources(roomResources=resources)
 
+    @Auth.user_roles('Admin')
+    def resolve_rooms_containing_resource(self, info, resource_id):
+        query = RoomResource.get_query(info).join(ResourceModel).join(
```

Comment:
Good work @rajeman . Kindly add validation when a non-existent resource id is provided